contributing_to_docs/create_or_edit_content.adoc

```diff
@@ -18,6 +19,24 @@ required to contribute
 * Read and review the link:doc_guidelines.adoc[Documentation guidelines] topic
 to understand the basic guidelines for consistency
 
+== Understanding the contribution workflow
+
+The following diagram outlines the steps required to add content to the OpenShift documentation:
+
+image::osdocs-contribution-workflow.png[OpenShift documentation contribution workflow]
+
+After you have identified a documentation requirement and created a ticket, you can contribute to the documentation directly or the OpenShift documentation team can create the content.
+
+When you contribute content directly, you must create a feature branch in a local clone of your own fork of the link:https://github.com/openshift/openshift-docs[openshift-docs repository]. After gathering stakeholder input and completing your technical testing, you can develop your documentation draft in your local feature branch. By using AsciiBinder on your local machine, you can create a local preview to review your updates.
+
+To submit your content for review, you must push your local feature branch to your openshift-docs fork on GitHub and create a PR. The PR is a request for the updates in your feature branch on your fork to be merged into the relevant branch in the mainstream openshift-docs repository. In the *Open a pull request* dialog, you can add a description, review the content updates, and create the PR. After your PR is created, the Netlify bot automatically generates a preview build and provides a preview link in a PR comment.
```

Comment:
For the first instance here of "feature branch", got me wondering if it should be a link to something like the following, in case the reader doesn't know what this means (vs "working branch" or just being unfamiliar with the process):

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches
^ actually uses the term "feature branch" verbatim

or

https://guides.github.com/introduction/flow/
^ I think might be a newer or at least friendlier intro?

It looks like throughout this doc, we've been calling them "feature branch", "local branch", and "working branch" interchangeably.

Reply:
Thank you for highlighting this. I have subsequently updated the references throughout the document to "feature branch" for consistency in the doc. I have also added a reference to the first of the links that you provided.